### PR TITLE
Only display the bonus when we've selected the ECS impact

### PR DESCRIPTION
**Pull request by magopian**

### CODE REVIEW

File: src/Page/Food/Builder.elm [hunk 1]
```diff
@@ -426,13 +426,17 @@ absoluteImpactView model results =
...
+                    div [ class "text-center fs-7" ]
+                        [ text " dont "
+                        , results.recipe.totalBonusesImpact.total
+                            |> Unit.impactToFloat
+                            |> Format.formatImpactFloat model.impact
+                        , text " de bonus inclus"
+                        ]
+
+                  else
+                    text ""
```
Comment (n1k0) [inline src/Page/Food/Builder.elm:439]: Sorry I landed something similar onto master already

File: src/Page/Food/Builder.elm [hunk 2]
```diff
@@ -1479,9 +1483,16 @@ stepResultsView model results =
         toFloat =
             Impact.getImpact model.impact.trigram >> Unit.impactToFloat
 
+        bonusToApply =
+            if model.impact.trigram == Impact.trg "ecs" then
+                Unit.impactToFloat results.recipe.totalBonusesImpact.total
+
+            else
+                0
+
```
Comment (n1k0) [inline src/Page/Food/Builder.elm:1492]: This si probably to be kept and landed though

**Review by n1k0 — APPROVED** [review]

**Comment by magopian** [comment]
I rebased on master, what do you think @n1k0 ?

**Comment by n1k0** [comment]
Perfect!

**Pull request merged**